src/functions.php

```diff
@@ -192,7 +192,7 @@ function task(string $name, $body = null): Task
  * @param string $task The task before $that should be run.
  * @param string|callable $do The task to be run.
  *
- * @return Task|void
```

Comment:
its invalid phpdoc to combine `void` with another type

Reply:
This is strange. Why func can't return something in one code path, and not return in another?

Reply:
its `Task|null` then and requires and explicit `return null`.

thats the opinion of phpstan though. in php-src its not necessary a technical requirement.

Reply:
Maybe it’s possible to allow Task|void? Because it’s valid php after all.

Reply:
feel free to discuss it with the phpstan maintainer :-)

https://github.com/phpstan/phpstan/issues/4014